src/gateways/DraftRegistrationGateway.ts

```diff
@@ -0,0 +1,8 @@
+import { DraftRegistration } from "../entities/DraftRegistration";
+
+export interface DraftRegistrationGateway {
+  deleteUse: (submissionId: string, useIndex: number) => Promise<void>;
+  createEmptyUse: (submissionId: string) => Promise<void>;
```

Comment:
Toyed with the idea of moving these to a `DraftBeaconUseGateway` to reduce coupling between the concepts of a `Use` and a `Registration`.  Continuing to mull it over.